src/GitUrlTagger.ts

```diff
@@ -51,6 +56,42 @@ export class GitUrlTagger implements IAspect {
     return ''; // .git directory not found
   }
 
+  findRootDirectory(): string {
+    let currentDir = process.cwd(); // Get the current directory
+
+
+    while (currentDir !== '/') {
+      const packageJson = path.join(currentDir, 'package.json');
+
+
+      if (fs.existsSync(packageJson)) {
+
+        return currentDir;
+      }
+
+      currentDir = path.dirname(currentDir); // Move up to the parent directory
+    }
+    return ''; // root directory not found
+  }
+
+  pullGitUrlFromFile() {
+    const rootpath = this.findRootDirectory();
+    const gitUrlTaggerConfig = path.join(rootpath, 'git-url-tagger.json');
+    if (fs.existsSync(gitUrlTaggerConfig)) {
+      const data = fs.readFileSync(gitUrlTaggerConfig, 'utf8');
+      const config = JSON.parse(data);
+      return config.url;
+    }
+  }
+
+  putGitUrlInFile(gitUrl: string) {
+    const rootpath = this.findRootDirectory();
+    console.log('rootpath: ' + rootpath);
+    const gitUrlTaggerConfig = path.join(rootpath, 'git-url-tagger.json');
```

Comment:
```suggestion
    const gitUrlTaggerConfig = path.join(rootpath, '.git-url-tagger.json');
```

Reply:
Also, since it's used in two places, please refactor the value into a const variable.

Reply:
Change implemented